documentation/Action-plugin-specification.md

```diff
@@ -110,28 +110,42 @@ The `action_plugin` table is the primary record of what actions are available to
 - `code`
 - `name`
 - `description`
-- `path` (determined by the folder the plugin was found in, plus the specified filename)
-- `function_name`
+- `path` (determined by the folder the plugin was found in)
 - `required_parameters`
 
 <a name="development"></a>
 
 ## Development and build process
 
-Plugins are intended to be stand-alone units, in that a user should be able to simply copy a plugin folder to another system and have it work straight away. (Eventually, we envisage that plugins could be imported directly via the front-end UI.) To that end, each plugin is developed as its own package, with its own `package.json` and `node_modules` folder, as well as Typescript and testing configurations.
+Plugins are intended to be stand-alone units, in that a user should be able to simply `yarn build` and copy content inside `build` folder to another system and have it work straight away. (Eventually, we envisage that plugins could be imported directly via the front-end UI.) To that end, each plugin is developed as its own package, with its own `package.json` and `node_modules` folder. Root typescript and testing configurations are used by default but can be overwritten
```

Comment:
It doesn't seem to work like this. I just tried running `yarn build` inside a plugin folder and it failed. Also -- when I just run `yarn install` within plugin folder, it says "already up-to-date", but there's no node_modules folder in there.

Reply:
Okay, it's working now (had to yarn install at top level). But build all still not working (see comment below -- wrong filename I think)

Reply:
> It doesn't seem to work like this. I just tried running yarn build inside a plugin folder and it failed. Also -- when I just run yarn install within plugin folder, it says "already up-to-date", but there's no node_modules folder in there.

Hmm, if root level node_modules are inline with root level package.json, should be able to `yarn build` from within plugin folder, I've just checked and it seems to work.

Reply:
Yeah, it works now (see prev comment).